nixos/modules/services/cluster/kubernetes/default.nix

```
@@ -25,8 +27,9 @@ let
         cluster = "local";
         user = name;
       };
-      current-context = "local";
+      name = name;
```

```suggestion
      inherit name;
```

also move it to the top.